Node >16 `TypeError`: Failed to parse URL from @ffmpeg/core/dist/ffmpeg-core.wasm

**Describe the bug**
When running the first example on the README in a NodeJS environment, I get the following error:

```js
❯ node --experimental-wasm-threads index.js
[info] use ffmpeg.wasm v0.11.4
[info] load ffmpeg-core
[info] loading ffmpeg-core

[fferr] TypeError: Failed to parse URL from /path/to/nodeFfmpeg/node_modules/@ffmpeg/core/dist/ffmpeg-core.wasm
/path/to/nodeFfmpeg/node_modules/@ffmpeg/core/dist/ffmpeg-core.js:12

if(l){ra=oa?require("path").dirname(ra)+"/":__dirname+"/";ta=function(a,b){wa||(wa=require("fs"));xa||(xa=require("path"));a=xa.normalize(a);return wa.readFileSync(a,b?null:"utf8")};va=function(a){a=ta(a,!0);a.buffer||(a=new Uint8Array(a));assert(a.buffer);return a};1<process.argv.length&&(ka=process.argv[1].replace(/\\/g,"/"));ja=process.argv.slice(2);process.on("uncaughtException",function(a){if(!(a instanceof ca))throw a;});process.on("unhandledRejection",u);la=function(a){process.exit(a)};f.inspect=
                                                                                                                           
                                                                                                                                                                                                                                                                                                         ^

RuntimeError: abort(TypeError: Failed to parse URL from /path/to/nodeFfmpeg/node_modules/@ffmpeg/core/dist/ffmpeg-core.wasm). Build with -s ASSERTIONS=1 for more info.
    at process.u (/path/to/nodeFfmpeg/node_modules/@ffmpeg/core/dist/ffmpeg-core.js:25:109
```


**To Reproduce**
Steps to reproduce the behavior:
1. Download the attach ZIP containing `package.json`, `package-lock.json`, `index.js` and the flame `test.avi`
    ==> [node-ffmpeg-error.zip](https://github.com/ffmpegwasm/ffmpeg.wasm/files/9498896/node-ffmpeg-error.zip)
2. `npm install` everything
3. `node --experimental-wasm-threads index.js`
4. See error

**Expected behavior**
It's the default example, it should just run.

**Desktop:**
 - OS: Darwin Kernel Version 21.6.0 arm64

**NodeJS:**
 - Version v18.8.0

**ffmpeg.wasm**
 - @ffmpeg/core version: 0.11.0  (latest release rn)
 - @ffmpeg/ffmpeg version: 0.11.4  (latest release rn)

**Additional context**
I wrote a lib that uses ffmpeg.wasm, and it works amazingly in the browser, but when running it on Node, despite my best efforts, I keep running into this.

I thought my setup was wrong so I created this minimalist NodeJS app that uses the README code, and lo and behold it shows the same error.

Here is the README code I'm referring to:

```js
const fs = require('fs');
const { createFFmpeg, fetchFile } = require('@ffmpeg/ffmpeg');

const ffmpeg = createFFmpeg({ log: true });

(async () => {
  await ffmpeg.load();
  ffmpeg.FS('writeFile', 'test.avi', await fetchFile('./test.avi'));
  await ffmpeg.run('-i', 'test.avi', 'test.mp4');
  await fs.promises.writeFile('./test.mp4', ffmpeg.FS('readFile', 'test.mp4'));
  process.exit(0);
})();
```

Downgrade to node 16.x still not work when deployed to Vercel.

`Runetime: Node.js 16.x`

```
[info] use ffmpeg.wasm v0.11.6
[info] load ffmpeg-core
[info] loading ffmpeg-core
Error: Cannot find module '@ffmpeg/core'
Require stack:
- /var/task/node_modules/@ffmpeg/ffmpeg/src/node/getCreateFFmpegCore.js
- /var/task/node_modules/@ffmpeg/ffmpeg/src/node/index.js
- /var/task/node_modules/@ffmpeg/ffmpeg/src/createFFmpeg.js
- /var/task/node_modules/@ffmpeg/ffmpeg/src/index.js
- /var/task/.next/server/pages/api/whisper.js
- /var/task/node_modules/next/dist/server/next-server.js
- /var/task/___next_launcher.cjs

```

--- a/src/createFFmpeg.js
+++ b/src/createFFmpeg.js
@@ -8,7 +8,7 @@ const NO_LOAD = Error('ffmpeg.wasm is not ready, make sure you have completed lo
 module.exports = (_options = {}) => {
   const {
     log: optLog,
-    logger,
+    logger: optLogger,
     progress: optProgress,
     ...options
   } = {
@@ -21,7 +21,7 @@ module.exports = (_options = {}) => {
   let runResolve = null;
   let runReject = null;
   let running = false;
-  let customLogger = () => {};
+  let customLogger = optLogger;
   let logging = optLog;
   let progress = optProgress;
   let duration = 0;
@@ -94,7 +94,7 @@ module.exports = (_options = {}) => {
    * In browser environment, the ffmpeg.wasm-core script is fetch from
    * CDN and can be assign to a local path by assigning `corePath`.
    * In node environment, we use dynamic require and the default `corePath`
-   * is `$ffmpeg/core`.
+   * is `@ffmpeg/core`.
    *
    * Typically the load() func might take few seconds to minutes to complete,
    * better to do it as early as possible.
@@ -115,6 +115,10 @@ module.exports = (_options = {}) => {
         wasmPath,
       } = await getCreateFFmpegCore(options);
       Core = await createFFmpegCore({
+        /*
+         * Avoid automatic exit after main is run
+         */
+        noExitRuntime: true,
         /*
          * Assign mainScriptUrlOrBlob fixes chrome extension web worker issue
          * as there is no document.currentScript in the context of content_scripts
@@ -141,7 +145,7 @@ module.exports = (_options = {}) => {
           return prefix + path;
         },
       });
-      ffmpeg = Core.cwrap(options.mainName || 'proxy_main', 'number', ['number', 'number']);
+      ffmpeg = Core.cwrap(options.mainName || '_emscripten_proxy_main', 'number', ['number', 'number']);
       log('info', 'ffmpeg-core loaded');
     } else {
       throw Error('ffmpeg.wasm was loaded, you should not load it again, use ffmpeg.isLoaded() to check next time.');
@@ -237,7 +241,7 @@ module.exports = (_options = {}) => {
       }
       running = false;
       try {
-        Core.exit(1);
+        Core.exit(0);
       } catch (err) {
         log(err.message);
         if (runReject) {

